source/manual/alerts/signon-api-user-token-expiry-soon.html.md

```diff
@@ -0,0 +1,68 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: Alertmanager SignonApiUserTokenExpirySoon
+parent: "/manual.html"
+layout: manual_layout
+section: Alertmanager alerts
+---
+
+[signon]: https://signon.publishing.service.gov.uk/api_users
+[gds-api-adapters]: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api.rb
+
+One or more tokens for API Users are about to expire. You should rotate
+expiring tokens to ensure the associated application keeps working.
+
+> If the tokens are
+> `New London: Trade Tariff Admin (PaaS) token for New London: Trade Tariff Backend (PaaS)`
+> see [Trade Tariff Admin on the Wiki](https://gov-uk.atlassian.net/wiki/spaces/PLOPS/pages/3155099649/Trade+Tariff+Admin)
+>
+> If the tokens are for the `Signon API Client (permission and suspension updater)`
+> you should contact the permissions team.
+
+As a working example, let's say we have an alert like *Content Publisher token
+for Publishing API expires in X days*. In this example, the API User is
+*Content Publisher* and the application is *Publishing API*.
```

Comment:
I don't think we need this working example - the new alert is explicit about what's the API User and what's the application.